en/resources/tutorials/templating.md

```diff
@@ -14,9 +14,7 @@ redirect_from: "/resources/templating.html"
 	<h1>Using Templating Engines with Kitura</h1>
 </div>
 
-Kitura is a lightweight, modular framework &mdash; this means that by default you get only basic HTTP functionality, like routing, parameter handling, parsing request body, etc. You can augment Kitura's out-of-the-box functionality by adding additional packages to your `Package.swift` file.
-
-Why use a template engine in your application? A [template engine](https://en.wikipedia.org/wiki/Template_processor) allows rendering of documents using static templates by substituting template variables with actual values at runtime. This means that you have clear separation between your template and data. You can also generate multiple web pages dynamically with a single template! This page explains how you can use template engines integrated with Kitura in your Kitura apps.
+Why use a template engine in your application? A [template engine](https://en.wikipedia.org/wiki/Template_processor) allows rendering of documents using static templates by substituting template variables with actual values at runtime. This means that you have clear separation between your template and data. You can also generate multiple web pages dynamically with a single template!
```

Comment:
Do we need the starting question "Why use a template engine in your application?
I might split the second sentence since it is a bit long to follow:
"A template engine allows rendering of documents using static templates. At runtime, template variables are substituted with actual values to generate an HTML file."

Reply:
We possibly don't need the question at the start, we do need a section that explains what a templating engine is and why you'd want to use it though and this section needs a title. 
The `"Why use a template engine in your application?` title persisted from the original doc, but maybe we should change this.